### PR TITLE
[GHA] Check if En_GBLocalise class exist in translations

### PR DESCRIPTION
**Pull request by wojsmol**
- Adds a comment to PR if triggered from a PR
- Replaces a PR comment content if comment already exists
- Can be triggered manually - in that case output is available in details of the workflow run for now

fix https://github.com/joomla/core-translations/issues/378

### CODE REVIEW

File: .github/workflows/_en-gb-localise.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Check existens of En_GBLocalise in translations
+
+on:
+  pull_request:
+    branches:
+      - main
+      - test-workflow
```
Comment (tecpromotion) [inline .github/workflows/_en-gb-localise.yml:7]:
```suggestion
```

We have no test-workflow here, right?

**Comment by wojsmol** [comment]
There is an issue if PR is created from repo fork - caused by default github token permission.

File: .github/workflows/_en-gb-localise.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    branches:
+      - main
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  engb-localise:
+    runs-on: ubuntu-latest
+    
+    permissions:
```
Comment (tecpromotion) [inline .github/workflows/_en-gb-localise.yml:14]: @wojsmol did we need the permissions?

We have not set permissions in any GHA.

**Review by tecpromotion — APPROVED** [review]
Many thanks @wojsmol 
Let's see this GHA in action 🚀

**Pull request merged**